tensorflow/contrib/bayesflow/python/kernel_tests/hmc_test.py

```diff
@@ -348,6 +348,82 @@ def testAIS2(self):
   def testAIS12(self):
     self._ais_gets_correct_log_normalizer_wrapper([1, 2])
 
+  def _skip_metropolis_step_correct(self, initial_draws, event_dims,
+                                    sess, feed_dict=None):
+    def log_gamma_log_prob(x):
+      return self._log_gamma_log_prob(x, event_dims)
+
+    step_size = array_ops.placeholder(np.float32, [], name='step_size')
+
+    if feed_dict is None:
+      feed_dict = {}
+
+    feed_dict[step_size] = 0.4
+
+    sample, acceptance_probs, _, _ = hmc.kernel(step_size, 5, initial_draws,
+                                                log_gamma_log_prob, event_dims)
+    skip_sample, skip_acceptance_probs, _, _ = hmc.kernel(
+      step_size, 5, initial_draws, log_gamma_log_prob, event_dims,
+      skip_metropolis_step=True)
+
+    # Need to set the same seed to make the velocity samples same
+    seed = np.random.randint(0, np.iinfo(np.uint32).max)
```

Comment:
Rather than making `seed` random, just choose some fixed integer.  That will prevent flakes in the rare event when you get an unlucky seed.